doc/Type/Cool.pod6

```diff
@@ -1163,12 +1163,18 @@ L<Regex|/type/Regex> as a list of strings.
 
     multi method contains(Cool:D: Str(Cool) $needle, Cool $start? --> Bool:D)
 
+Coerces the first two arguments (in method form, also counting the invocant)
```

Comment:
I copied from a nearby source. As far as I can tell, it's not yet explained well enough that I can be sure what it means, so I've got to go a little cargo cult and trust that the other answers are correct.